examples/route_dependencies/stac-realm.json

```diff
@@ -0,0 +1,1847 @@
+{
```

Comment:
If you could explain this too. This is clearly used with keycloak. It's a fairly long file ... but it's probably not a problem - it's only 67 kb.

Reply:
It's a saved copy of a test keycloak with a realm and one user already created. I could remove this and just add the steps to create a realm and user to the readme if that's preferable?

Reply:
It's nice to have to fire off the docker compose example quickly. It's not being packaged in our pypi release so I don't think the size matters. We could add the steps to create a realm in the readme which would probably be really good.

Reply:
I created an issue on adding documentation for creating a keycloak file